### DISCUSSION TITLE
Suggested names for devices

### DISCUSSION THREAD

**Discussion by apbarratt** [john30/ebusd#984] [Q&A]
Hi guys, I'm wondering if anyone can provide some suggested names for the devices that my ebusd is reporting in Home Assistant?

I've got a Vaillant Heatpump running a wet central heating system and hot water with a buffer tank in the attic.

The devices that show up are `hmu` and `basv`.  Seemingly these correspond to the files `08.hmu.csv` and `15.basv.csv`.

For now I've named `hmu` to be `heatpump manager` and `basv` to be `heatpump` but I'm not really sure these properly reflect what the devices are and what they do.

`hmu` seems to show all sorts to do with power usage and modes, with stuff changing to reflect when I boost hot water from the thermostat, etc.  It also shows two temperatures, I think for input and output flow?

`basv` on the other hand seems to hold data about the house more, such as hot water, room, and outdoors temperatures.  Curiously though it doesn't seem to include the humidity that the thermostat shows.

I'm just hoping someone could suggest good names for them so I can build a better picture in my head of what these devices are :)

**Comment by JonesGW-MGD** [comment]
Maybe 7 months too late, but an 0.8 is a heat source so something like a  boiler but in your case the heat pump, the 'basv' device is the controller - so something like a SENSOComfort, SENSOHome, vSMART device etc.
